openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1788,6 +1895,10 @@ The following is a non-normative example of a Credential Offer of type `ldp_vc`:
 
 <{{examples/credential_offer_ldp_vc.json}}
 
+For better suiting the W3C Data Integrity Proof ecosystem the following non-normative example displays a Credential request with the proof of possession type set to `ldp_vp`, using a verifiable presentation.
+
+<{{examples/credential_request_ldp_vc_vp.json}}
+
```

Comment:
This is a section about credential offer, but example is about credential request. Please put this example in `#### Credential Request {#credential_request_ldp_vc}` section.

Reply:
https://github.com/openid/OpenID4VCI/pull/66/commits/31051475939b42d2cb2e93611f6030d15522e7b8